src/librustc_driver/driver.rs

```diff
@@ -1007,15 +1014,15 @@ pub fn phase_4_translate_to_llvm<'a, 'tcx>(tcx: TyCtxt<'a, 'tcx, 'tcx>,
     let translation =
         time(time_passes,
              "translation",
-             move || trans::trans_crate(tcx, &mir_map, analysis));
+             move || trans::trans_crate(tcx, &mir_map, analysis, &hashes_map));
 
     time(time_passes,
          "assert dep graph",
          move || rustc_incremental::assert_dep_graph(tcx));
 
     time(time_passes,
          "serialize dep graph",
-         move || rustc_incremental::save_dep_graph(tcx));
+         move || rustc_incremental::save_dep_graph(tcx, &hashes_map));
```

Comment:
I'd prefer a less generic name than `hashes_map` here and above. `incr_comp_hashes` or `ic_hashes_map` or `ich_map`, something that makes it visible that this has to do with incr. comp.